pronto/api/checks/annotations.py

```diff
@@ -16,14 +16,13 @@
 
 
 def ck_begin_uppercase(cabs: LoT, exceptions: DoS) -> Err:
-    prog = re.compile(r"^\s*(?:<p>)?\s*([a-z])")
+    prog = re.compile(r"^\s*(?:<p>)?\s*([a-z][A-Za-z]+)\s+")
```

Comment:
- Replacing `[a-z][A-Za-z]+` by `[a-z][A-Za-z]*`: the former won't match _a family_
- I assume the `\s+` is to find the end of the first word. In this case, [`\b`](https://docs.python.org/3/library/re.html#index-28) might be a better choice.

```suggestion
    prog = re.compile(r"^\s*(?:<p>)?\s*([a-z][A-Za-z]*)\b")
```